### PR TITLE
May newsletter 2020

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

File: _posts/2020-06-01-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+These are some of the bigger things we've done over the last month.
+
+### Fastly Cost Saving
+
+This month we updated our Fastly configuration for the Origami Build Service and Origami Image Service. Our new Fastly contract meant that our previous architecture was no longer good from a cost perspective (it used to be fine with our old contract).
+
+With the update we're saving ~£5,000 per month.
+
+Shout out to Sam Parkinson for his help with this.
```
Comment (JakeChampion) [inline _posts/2020-06-01-newsletter.md:25]:
```suggestion
Thank you Sam Parkinson for calculating the cost saving and helping this work get prioritised.
```

File: _posts/2020-06-01-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+obt develop
+```
+
+If you've never used Origami Build Tools before and would like to understand what this command does, [see the Origami Build Tools readme](https://github.com/Financial-Times/origami-build-tools#usage).
+
+#### Speed Improvements
+
+Origami Build Tools now compiles component code faster. It's an important metric for Origami Build Tools as it helps keep the feedback loop short between an engineer making a code change and seeing the result of that change.
+
+As an unscientific example on my work laptop: the quickest of three [o-table](https://github.com/Financial-Times/o-table) demo builds is 2.18 seconds over 4.11 seconds -- around a 53% improvement.
```
Comment (JakeChampion) [inline _posts/2020-06-01-newsletter.md:67]:
```suggestion
As an unscientific example on an FT MacBook Pro laptop: the quickest of three [o-table](https://github.com/Financial-Times/o-table) demo builds is 2.18 seconds over 4.11 seconds -- around a 53% improvement.
```

File: _posts/2020-06-01-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+We improved the performance of Origami Build Tools by creating two new packages:
+- [Financial-Times/scrumple](https://github.com/Financial-Times/scrumple#scrumple): a fast (and scrappy) JavaScript bundler for developing Origami components, based on a now-deleted tool called Pax.
+- [Financial-Times/sass](https://github.com/Financial-Times/sass): Prebuilt [dart-sass](https://github.com/sass/dart-sass) binaries available via NPM.
+
+## Special Thanks
+
+This month our special thanks goes to Rowan Manning! In case you missed the news, at the start of May Rowan moved from the Origami team (O&R) to Customer Products. So a big thank you to Rowan for all their work on and leading Origami over the years 😃 (And congratulations again to Jake Champion, who is our new Origami tech lead!)
```
Comment (JakeChampion) [inline _posts/2020-06-01-newsletter.md:75]: Let's keep it to only thanking Rowan, I don't need any congratulations in the Origami Newsletter :-)
```suggestion
This month our special thanks goes to Rowan Manning! In case you missed the news, at the start of May Rowan moved from the Origami team (O&R) to Customer Products. So a big thank you to Rowan for all their work on and leading Origami over the years 😃
```

File: scripts/project-board-summary.js [hunk 4]
```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env node
```
Comment (JakeChampion) [inline scripts/project-board-summary.js:1]: We should add a section about this tool to the [Authoring and sending a newsletter](https://github.com/Financial-Times/origami-website/tree/2020-06-01-newsletter#authoring-and-sending-a-newsletter) documenation

File: _posts/2020-06-01-newsletter.md (hunk 3 above)
Comment (notlee) [reply _posts/2020-06-01-newsletter.md:75]: I thought people who haven't heard might find it useful to know who the team lead is now, and thought this was a nice way to include that info :) maybe a reword? happy to remove if you'd prefer

**Comment by chee** [comment]
this is good

File: _posts/2020-06-01-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+## Broader Update
+
+A digest of other things that have happened since our last update:
+
+- MAJOR: [Financial-Times/origami-bower-registry](https://github.com/Financial-Times/origami-bower-registry). Upgrade to node 12, drop support for previous versions.
+- NEW: [Financial-Times/origami-workshop](https://github.com/Financial-Times/origami-workshop): We created a small tool to help attendees of the Intro to Origami workshop complete [the manual build tutorial](https://origami.ft.com/docs/tutorials/manual-build/). The tool helps attendees focus on on building projects with Origami components, instead of setting up build tools which may differ depending on their team.
```
Comment (notlee) [inline _posts/2020-06-01-newsletter.md:82]:
```suggestion
- NEW: [Financial-Times/origami-workshop](https://github.com/Financial-Times/origami-workshop): We created a small tool to help attendees of the Intro to Origami workshop complete [the manual build tutorial](https://origami.ft.com/docs/tutorials/manual-build/). The tool helps attendees focus on building projects with Origami components, instead of setting up build tools which may differ depending on their team.
```

File: _posts/2020-06-01-newsletter.md (hunk 3 above)
Comment (JakeChampion) [reply _posts/2020-06-01-newsletter.md:75]: That makes much more sense now :-)
I'd reword it to not have the congratulations, something like this?
```suggestion
This month our special thanks goes to Rowan Manning! In case you missed the news, at the start of May Rowan moved from the Origami team (O&R) to Customer Products. So a big thank you to Rowan for all their work on and leading Origami over the years 😃. Jake Champion is the stand-in Tech Lead, if you have questions about Origami please contact them instead of Rowan :-)
```
